catalog/task/apply-mapping/0.2/apply-mapping.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: apply-mapping
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to apply a mapping to a Snapshot
+  params:
+    - name: snapshot
+      type: string
+      description: The Snapshot in JSON format to apply the mapping to
+    - name: extraConfigPath
+      type: string
+      description: The path to the config file containing the mapping
+      default: ""
+  workspaces:
+    - name: config
+      description: The workspace where the extra config file containing the mapping resides
+  results:
+    - name: snapshot
+      description: |
+        The AppicationSnapshot in JSON format with the mapping applied. If there is no mapping key in the
```

Comment:
description is wrong. Snapshot not ApplicationSnapshot

Reply:
nice catch